ray-operator/controllers/ray/rayjob_controller.go

```diff
@@ -888,3 +893,10 @@ func checkActiveDeadlineAndUpdateStatusIfNeeded(ctx context.Context, rayJob *ray
 	rayJob.Status.Message = fmt.Sprintf("The RayJob has passed the activeDeadlineSeconds. StartTime: %v. ActiveDeadlineSeconds: %d", rayJob.Status.StartTime, *rayJob.Spec.ActiveDeadlineSeconds)
 	return true
 }
+
+func emitRayJobMetrics(originalRayJobInstance, rayJobInstance *rayv1.RayJob) {
+	// If a RayJob was in `New` status and is now in `Initializing`, it means the RayJob has been created.
```

Comment:
Do you think it might be better to place the `enableMetrics` if branch here?
My concern is, there could be multiple places changing ray job status, having a centralized place to decide emit or not helps simplify logic.